uwblueprint/jump-math
# feat: test sessions page

## Notion ticket link

[Test Sessions page](https://www.notion.so/uwblueprintexecs/Test-Sessions-page-da4a9b79d4c44a7781088909135d14d8)



## Implementation description
<img width="1412" alt="image" src="https://github.com/uwblueprint/jump-math/assets/9922514/a4849557-f448-4c19-9411-69aa12683068">




## Steps to test
1. Comment out parts of the code/change the error/loading states to see the different UI states.



## What should reviewers focus on?
* 


## Checklist
